console/console-init/ui/src/components/TypeAheadSelect/TypeAheadSelect.tsx

```diff
@@ -52,6 +52,7 @@ const TypeAheadSelect: React.FunctionComponent<ITypeAheadSelectProps> = ({
         <SelectOption
           value={TypeAheadMessage.MORE_CHAR_REQUIRED}
           isDisabled={true}
+          key="2"
```

Comment:
Can we rename the key to be a bit informative?

Reply:
yes, I'll do it in next PR.